data/hlint.yaml

```diff
@@ -222,7 +224,9 @@
     - warn: {lhs: any id, rhs: or}
     - warn: {lhs: all id, rhs: and}
     - warn: {lhs: any (not . f) x, rhs: not (all f x), name: Hoist not}
+    - warn: {lhs: any (notElem y) x, rhs: not (all (elem y) x), name: Hoist not}
```

Comment:
I'd say these are more confusing in the replacement, and no more efficient, so better to leave them as is.